infrastructure/terraform/components/app/amplify_domain_association.tf

```diff
@@ -4,13 +4,13 @@
 #   enable_auto_sub_domain = true
```

Comment:
Should these 4 infrastructure changes be here? They seem unrelated to component tests.

Reply:
These are rebased from the main branch, don't know why it came as changes on this PR but they are present in main and is being worked on by someone else so might leave it.

Reply:
They are not changes related to this ticket so shouldn't be in this PR.

Reply:
I think this goes back to my question as to why you are merging this into CCM-5628. It increases the chances of things getting messy.

Reply:
Merging main into CCM-5628 might clean things up in this PR.